source/MaterialXGenShader/CMakeLists.txt

```diff
@@ -1,13 +1,38 @@
 file(GLOB_RECURSE materialx_source "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
 file(GLOB_RECURSE materialx_headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h*")
 
+if(NOT MATERIALX_BUILD_OCIO)
+    list(REMOVE_ITEM materialx_source "${CMAKE_CURRENT_SOURCE_DIR}/OCIOColorManagementSystem.cpp")
+endif()
+
 assign_source_group("Source Files" ${materialx_source})
 assign_source_group("Header Files" ${materialx_headers})
 
 add_library(MaterialXGenShader ${materialx_source} ${materialx_headers})
 
 add_definitions(-DMATERIALX_GENSHADER_EXPORTS)
 
+target_link_libraries(
+    MaterialXGenShader
+    MaterialXCore
+    MaterialXFormat
+    ${CMAKE_DL_LIBS})
+
+if(MATERIALX_BUILD_OCIO)
+    set(MATERIALX_OCIO_DIR ${MATERIALX_OCIO_DIR})
+    message(STATUS "OCIO root path: ${MATERIALX_OCIO_DIR}")
+
+    message(STATUS "OCIO include path: ${MATERIALX_OCIO_DIR}/include")
+    include_directories("${MATERIALX_OCIO_DIR}/include")
+ 
+    message(STATUS "OCIO library path: ${MATERIALX_OCIO_DIR}/lib")
+    file(GLOB OCIO_LIBRARIES "${MATERIALX_OCIO_DIR}/lib/*.lib") # ${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
For now use hard-coded paths vs a "find package" utility as none currently exists.
Todo: fix up library dependencies to use proper platform extensions.